docs/tr2/README.md

```diff
@@ -93,6 +93,20 @@ game with new enhancements and features.
     - fixed being unable to stack multiple pushblocks over multiple rooms
     - fixed falling pushblocks using the enemy grunt sound effect
 - fixed seaweed collision in Living Quarters preventing Lara from climbing out of the water in room 15
+- fixed the scale and rotation of several pickup models:
+    - increased auto pistol ammo size
+    - increased M16 ammo size
+    - increased grenade size
+    - reduced offshore key card sizes, and fixed inventory rotation
```

Comment:
`Offshore Rig`

Reply:
Thanks, fixed. Was meaning for it to cover the two levels, but have made it explicit instead.